src/main/kotlin/com/julianasaran/blog/application/list/ListAuthorsQuery.kt

```diff
@@ -0,0 +1,5 @@
+package com.julianasaran.blog.application.list
+
+class ListAuthorsQuery(val name: String) {
```

Comment:
Since name might not be provided, the value could be nullable
```suggestion
class ListAuthorsQuery(val name: String) {
```